api/src/main/java/org/openmrs/module/openconceptlab/UpdateService.java

```diff
@@ -258,4 +261,32 @@ public void scheduleUpdate() {
 			scheduler.schedule(subscription);
 		}
 	}
+
+	/**
+	 *
+	 * @param update the update to be passed
+	 * @param first starting index
+	 * @param max  maximum limit
+	 * @return a list of items
+	 */
+	public List<Item> getUpdateItems(Update update, int first, int max) {
+		List<Item> list = new ArrayList<Item>(update.getItems());

```

Comment:
@rkorytkowski if you have time plaese help me understand how to link that criteria with already existing Update object